TUM Campus App/LecturesTableViewController.swift

```diff
@@ -70,7 +70,7 @@ class LecturesTableViewController: RefreshableTableViewController<Lecture>, Deta
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let item = lectures[indexPath.section].1[indexPath.row]
-        let cell = tableView.dequeueReusableCell(withIdentifier: item.getCellIdentifier()) as? CardTableViewCell ?? CardTableViewCell()
+        let cell = tableView.dequeueReusableCell(withIdentifier: item.getCellIdentifier()) ?? CardTableViewCell()
```

Comment:
We don't need the `?? CardTableViewCell()` if we use `dequeueReusableCell(withIdentifier:forIndexPath:)`